Makefile

```diff
@@ -199,6 +199,12 @@ include $(RULESPATH)/rules.mk
 # Custom rules
 #
 
+clang-format:
+	clang-format --style=LLVM -i *.c ./board/*.{c,h} ./source/*.{c,h}
+
+clang-format-ci:
```

Comment:
added 2 targets. One for CI, one for manual run. Did not add `@` to make it verbose.